microsoft-edge/extensions/publish/marketing-guidelines.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Marketing guidelines for Microsoft Edge Add-ons
+description: Marketing guidelines for publishing an extension, theme, or web app, such as using badges, at Microsoft Edge Add-ons.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 08/13/2025
+---
+# Marketing guidelines for Microsoft Edge Add-ons
+<!-- todo: is this content copied (then modified) from a webpage? which? -->
+
+<!-- todo: global in this article: lexicon:
+the Microsoft Edge Add-on badge
+the Microsoft Edge add-on badge
+the Microsoft Edge add-ons badge
+the Microsoft Edge Add-ons badge
+the Featured badge
+-->
+
+<!-- todo: global in Extensions docs: which?
+add-ons (such as extensions, themes, or web apps)
+add-ons (such as extensions, themes, or apps)
+-->
+
+These guidelines outline how to use Microsoft Edge brand assets (such as badges and logos) across websites, documentation, tutorials, and other services or integrations.<!-- todo: ", especially the [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) site"?  is this article specifically for the Microsoft Edge Add-ons site, or other sites as well? -->  They ensure consistent, accessible, and compliant representation of the Microsoft Edge Add-ons brand.
```

Comment:
"and logos" - currently this article has almost no points regarding logos - strike "and logos?"

@G-u-r-j-e-e-t